KachnaOnline.App/appsettings.json

```diff
@@ -1,3 +1,27 @@
 {
-
+  "ConnectionStrings": {
+    "AppDb": ""
+  },
+  "Kis": {
+    "ApiBaseUrl": "https://su-int.fit.vutbr.cz/kis/api/",
+    "DisplayToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdS1pbnQua2lzLnZ1dGJyLmN6IiwiaWF0IjoxNjMzNTEwMTU5LCJleHAiOjE2NDkwNjIxNTksInN1YiI6IjE5NSIsInNjcCI6ImdkIn0.9ZGxU_mH9bMJ9tmj7E6ShKIVxW1xvMIDMWikjaIMlpc",
```

Comment:
What is this token for? Is it some kind of a public token? Otherwise it probably shouldn't be here, should it?